File: src/Subsystems/Climber.cpp
#include "Climber.h"
#include "../RobotMap.h"

Climber::Climber() : Subsystem("Climber") {
	climberMotor = new VictorSP(climbMotor);
}

void Climber::InitDefaultCommand() {
	// Set the default command for a subsystem here.
	// SetDefaultCommand(new MySpecialCommand());
}

// Put methods for controlling this subsystem
// here. Call these from Commands.
void Climber::Set(double value){
	climberMotor->Set(value);
}

File: src/org/usfirst/frc/team9159/robot/subsystems/Climber.java
package org.usfirst.frc.team9159.robot.subsystems;

import edu.wpi.first.wpilibj.command.Subsystem;
import org.usfirst.frc.team9159.robot.RobotMap;
import org.usfirst.frc.team9159.robot.commands.Climb;

import edu.wpi.first.wpilibj.VictorSP;

public class Climber extends Subsystem {
	  private VictorSP climberMotor;
	  private VictorSP climberMotor2;
	  public Climber() {
		    climberMotor = new VictorSP(RobotMap.CLIMBER_MOTOR);
		    climberMotor2 = new VictorSP(RobotMap.CLIMBER_MOTOR_2);
	  }
	  public void initDefaultCommand(){
		  setDefaultCommand(new Climb());
	  }
	  public void set(double speed) {
	    climberMotor.set(-speed);
	    climberMotor2.set(-speed);
	  }
}